src/Sarif/CodeGenHints.json

```diff
@@ -314,40 +314,6 @@
       }
     }
   ],
-
-  "Rule.Configuration": [
-    {
-      "kind": "EnumHint",
-      "arguments": {
-        "typeName": "RuleConfiguration",
-        "description": "A value that specifies whether a rule is enabled.",
-        "memberNames": [
-          "Unknown",
-          "Enabled",
-          "Disabled"
-        ]
-      }
-    }
-  ],
-
-  "Rule.DefaultLevel": [
-    {
-      "kind": "EnumHint",
-      "arguments": {
-        "typeName": "ResultLevel",
-        "description": "The level of a result.",
-        "zeroValueName": "Default",
-        "memberNames": [
-          "NotApplicable",
-          "Pass",
-          "Note",
-          "Warning",
-          "Error"
-        ],
-        "allowMemberCountMismatch": true
-      }
-    }
-  ],
```

Comment:
This should not go away. Instead, it should become the type of the new `RuleConfiguration.DefaultLevel` property (which, without this declaration, would be a string).

Reply:
Fixed